modules/bootstrap/variables.tf

```diff
@@ -1,46 +1,46 @@
-variable "resource_group_name" {
-  description = "Name of the Resource Group to use."
-  type        = string
-}
-
-variable "location" {
-  description = "Region to deploy vm-series bootstrap resources. Ignored when using an `existing_storage_account`."
-  default     = null
-  type        = string
-}
-
 variable "create_storage_account" {
-  description = "If true, create a Storage Account and ignore `existing_storage_account`."
+  description = "If `true`, create a Storage Account."
   default     = true
   type        = bool
 }
 
 variable "storage_account_name" {
```

Comment:
It would be nice to add a `validation` block here

Reply:
👍🏻 and tested